lib/heroku/kensa/check.rb

```diff
@@ -281,9 +282,11 @@ def call!
           :heroku_id => heroku_id,
           :plan => data[:plan] || 'test',
           :callback_url => callback,
+	  :log_input_url => "https://token:t.01234567-89ab-cdef-0123-456789abcdef@1.us.logplex.io/logs",
```

Comment:
It looks like the description in that link creates a structure(in the manifest) of
```{
  "api": {
    "requires": [
      "log_input"
     ]
   }
}```
Is that correct?  I'm new to add ons, so I've never seen it before.